src/main/java/net/sf/jabref/sql/DBStringsPreferences.java

```diff
@@ -0,0 +1,55 @@
+package net.sf.jabref.sql;
+
+import net.sf.jabref.JabRefPreferences;
+
+import java.util.Arrays;
+import java.util.List;
+
+public final class DBStringsPreferences {
+
+    private final String serverType;
+    private final String serverHostname;
+    private final String username;
+    private final String database;
+
+    public DBStringsPreferences(String serverType, String serverHostname, String username, String database) {
+        this.serverType = serverType;
+        this.serverHostname = serverHostname;
+        this.username = username;
+        this.database = database;
+    }
+
+    public String getServerType() {
```

Comment:
I think it is better to change the return type to `DatabaseType` instead of the string value.